federatedscope/cv/dataset/leaf_cv.py

```diff
@@ -99,9 +97,9 @@ def __getitem__(self, index):
             index (int): Index
 
         Returns:
-            dict: {'train':[(image, target)], 
-                   'test':[(image, target)], 
-                   'val':[(image, target)]} 
+            dict: {'train':[(image, target)],
+                   'test':[(image, target)],
+                   'val':[(image, target)]}
```

Comment:
I cannot figure out the purpose of this method. It looks very strange to transform the whole dataset in a getitem method.